Flagsmith.FlagsmithClient/AnalyticFlag.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FlagsmithEngine.Feature.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+
+namespace Flagsmith
+{
+    public class AnalyticFlag : Flag
```

Comment:
Is there a reason that there is this additional class here? The python client handles the analytics processing on the `Flags` class which I don't think exists in your .NET implementation. Is there a specific reason for this?

Reply:
I have as per python implementation.